### PR TITLE
Update human clinical and health data sources page

### PR DESCRIPTION
**Pull request by IrisVanDam**
As we were encouraged to get familiar with GitHub I'm trying to add a first few lines of text to the human clinical and health data sources page. This is a first draft to try everything out, but hopefully it's a start.

### CODE REVIEW

**Comment by rabuono** [comment]
Great start!
Might be best to move the PR to draft until the page has enough content to be published.

**Comment by lauportell** [comment]
Closes #109

File: human_clinical_and_health_data/data_sources.md [hunk 1]
```diff
@@ -1,63 +1,142 @@
...
+* Cross-border health data exchange: The European Union is working to establish a common framework for cross-border exchange of health data: [The European Health Data Space (EHDS)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_en). This initiative aims to facilitate the exchange of health data across borders while ensuring data protection and privacy. The EHDS will include a set of technical standards and protocols to enable interoperability of health data across Europe. Currently the European Genome-phenome Archive (EGA) is expanding into a federation of nodes through the [Federated EGA](https://ega-archive.github.io/FEGA-onboarding/). FEGA enables deposition and distribution of health data that cannot be stored outside of the country of origin.
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:50]: Here I think you can make FEGA and EHDS different bullet points. 

For the EHDS, you could add that one of the key things it aims to do is improve findability of data through metadata catalogues using metadata standards.

File: human_clinical_and_health_data/data_sources.md [hunk 2]
```diff
@@ -1,63 +1,142 @@
...
 
-### Standards, schemas, ontologies and vocabularies 
+### Considerations
 
-<!--- (optional) --->
+* When selecting data sources for your research, it is important to consider their **reliability, relevance**, and **completeness**. Are the data sources trustworthy? Are they up-to-date and relevant to the infectious disease in question? Are the data sources complete, or are there gaps that need to be taken into account?
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:54]:
```suggestion
* When selecting data sources for your research, it is important to consider their **reliability, relevance**, and **completeness**. Are the data sources trustworthy and up-to-date? Are they relevant to the infectious disease in question and do they have the right data to answer your research question? Are the data sources complete, or are there gaps that need to be taken into account?
```

File: human_clinical_and_health_data/data_sources.md [hunk 3]
```diff
@@ -1,63 +1,142 @@
...
+* **Different data holders for different data sources**
+  * Human clinical and health data are collected as:
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:56]:
```suggestion
  * Human clinical and health data can be collected in the form of:
```

File: human_clinical_and_health_data/data_sources.md [hunk 4]
```diff
@@ -1,63 +1,142 @@
...
+      * Electronic health records
+      * Hospital data
+      * Pharmacy data
+      * Laboratory and test result data
+      * Disease registries
+      * Insurance claim data
+      * Mortality data
+      * Survey data
+      * Consumer data
+  * Depending on how the data are collected, the data can be found across different sources and data holders:
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:66]:
```suggestion
  * Depending on how the data are collected, the data can be found across different sources and data holders, for example:
```

File: human_clinical_and_health_data/data_sources.md [hunk 5]
```diff
@@ -1,63 +1,142 @@
...
+      * National Institutes of Public Health
+      * Electronic health records at hospitals or other healthcare providers
+      * The World Health Organization
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:69]:
```suggestion
      * International organisations (e.g., ECDC, WHO)
```

File: human_clinical_and_health_data/data_sources.md [hunk 6]
```diff
@@ -1,63 +1,142 @@
...
+      * Research data networks and consortia
+* The different organisation of human clinical and health data across European countries:
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:71]:
```suggestion
* Human clinical and health data is organised differently across different European countries:
```

File: human_clinical_and_health_data/data_sources.md [hunk 7]
```diff
@@ -1,63 +1,142 @@
...
 
-List of standards, schemas, ontologies and/or vocabularies that are relevant to the resources in this category. Consider having links to the "Data description" page.
-When applicable, consider adding the appropriate FAIRsharing entry to the tools and resources spreadsheet so that they can be listed and appropriately tagged at the end of the page
+### Considerations
 
-### Licenses 
+* **Process for access - figuring out the access procedures**
 
-<!--- (optional) --->
+  Generally, access to sensitive data is  granted on a request basis, that is linked to the secondary purposes (a purpose that is different from the original for which the data were collected) of a specific research project. The research protocol of the research project that is requesting data access is generally evaluated by a research ethics committee (REC) or a Data Protection Agency (DPA).
```
Comment (ShonaCosgrove) [inline human_clinical_and_health_data/data_sources.md:107]:
```suggestion
  Generally, access to sensitive data is  granted on a request basis, that is linked to the secondary purposes (a purpose that is different from the original for which the data were collected) of a specific research project. The research protocol of the research project that is requesting data access is generally evaluated by a research ethics committee (REC) and/or a Data Protection Agency (DPA).
```

**Review by ShonaCosgrove — COMMENTED** [review]
Overall, I think the page has a lot of good and useful information! I just found the structure a bit confusing and felt that it jumps back and forth between topics a bit. 

Maybe you could change the structure to: 
1. Introduction
2. Organisation of health data: including commonalities and differences between countries 
3. Finding (meta)data: including metadata catalogeues, RIs etc
4. Accessing data: including the information you have under access now, and move the regulatory aspects here

Essentially just adding a section 'Organisation' and moving some of the information from findability to access. I made some other small suggestions to wording. Happy to look at it together as well :)

File: human_clinical_and_health_data/data_sources.md [hunk 8]
```diff
@@ -1,63 +1,142 @@
...
+* Health data repositories: in some countries, national or regional health data repositories have been established to aggregate electronic health records across multiple sources. For example, in Sweden, [the National Patient Register](https://www.socialstyrelsen.se/en/statistics-and-data/registers/national-patient-register/) collects data from all healthcare providers in the country; while in the UK, [the National Health Service (NHS) Digital](https://digital.nhs.uk/) collects data from various healthcare settings. However, in many countries these types of registries are not yet in place.
+* Health research databases: in a few countries, national health research databases have been established to support scientific research. These databases typically include de-identified data from large patient populations.
```
[the remark this answers is not in the document]
Comment (IrisVanDam) [reply human_clinical_and_health_data/data_sources.md:48]: Well, finding good examples seems to be the issue here. Online I found that health research databases should exist, but it all remained rather vague.  I had hoped that by putting it out here someone might be able to help. For now I have removed it. If someone has any ideas we can always add it back on later :).

File: human_clinical_and_health_data/data_sources.md [hunk 9]
```diff
@@ -1,63 +1,145 @@
...
+    url: https://fairsharing.org/FAIRsharing.dj8nt8
+  - name: European Health Information Portal
+    url: https://doi.org/10.25504/FAIRsharing.8690f1
+
 # More information on how to fill in this metadata section can be found here https://www.infectious-diseases-toolkit.org/contribute/page_metadata
 ---
 
 ## Introduction
 
-## What is considered as human clinical and health data?
```
Comment (lauportell) [inline human_clinical_and_health_data/data_sources.md:24]: Ccan this be changed to "What is cosidered as human clinical and health data and why is it important for infectious diseases research?" to align with the other pages?

File: human_clinical_and_health_data/data_sources.md [hunk 10]
```diff
@@ -1,63 +1,145 @@
...
+* **Cross-border health data exchange**: the European Union is working to establish a common framework for cross-border exchange of health data: [The European Health Data Space (EHDS)](https://health.ec.europa.eu/ehealth-digital-health-and-care/european-health-data-space_en). This initiative aims to facilitate the exchange of health data across borders while ensuring data protection and privacy. The EHDS will include a set of technical standards and protocols to enable interoperability of health data across Europe. Currently the European Genome-phenome Archive (EGA) is expanding into a federation of nodes through the [Federated EGA](https://ega-archive.github.io/FEGA-onboarding/). FEGA enables deposition and distribution of health data that cannot be stored outside of the country of origin.
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+## Finding (meta)data
```
Comment (lauportell) [inline human_clinical_and_health_data/data_sources.md:54]: Do you think this section can be changed to "Search and discoverability" to align with other data sources pages?

**Comment by bedroesb** [comment]
@elixir-europe/idtk-editors This PR is ready for review! Can an editor volunteer for this page to review it?

**Review by lauportell — APPROVED** [review]

**Pull request merged**